src/styles/blocks/catalog.scss

```diff
@@ -12,14 +14,21 @@
     align-items: center;
     box-sizing: border-box;
 
-    width: 200px;
-    height: 300px;
+    width: $card-width;
+    height: $card-height;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+    transition-property: all;
```

Comment:
Don't use all. It's not good for performance. Better specify the exact properties you want add a transition to
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x